ffi/SDL2_0.lua

```diff
@@ -117,7 +117,8 @@ function S.waitForEvent(usecs)
 		if ffi.os == "OSX" and (event.type == SDL.SDL_FINGERMOTION or
 			event.type == SDL.SDL_FINGERDOWN or
 			event.type == SDL.SDL_FINGERUP) then
-			-- no op
+			-- noop for trackpad finger inputs which interfere with emulated mouse inputs
+			do end
```

Comment:
Or that. :-)

Reply:
Hah, it caught you! ;-D

```
    ffi/SDL2_0.lua:121:4: empty do..end block
```

Like I said, just stick a `-- luacheck: ignore 542` after the then. Or just writing return true or nil should do the trick as well.

Reply:
Yeah, seems `luacheck` does not buy it.